source/sass/global/global.scss

```
@@ -0,0 +1,26 @@
+a {
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;
+  vertical-align: middle;
+}
```

На эти дефолты тоже не очень хорошо завязываться. Лучше всё таки контролировать эти стили на уровне блоков. Для ссылок вообще по хорошему нужен блок `.link`. А изображения прилетающие из админок лучше контролировать в блоке `.content` одним из вложенных селекторов.
```suggestion
.link {
  text-decoration: none;
}

.content img {
  max-width: 100%;
  vertical-align: middle;
}
```

---

ммм, т.е. по хорошему надо везде где может быть генерируемый контент миксовать блок ```content``` ?

---

По хорошему да. Но сейчас сильно об этом не думай. Просто раз уже хотел как-то обезопасить вёрстку от случайно запихнутого хайреза, то надо хотя бы это сделать грамотно. Потом дополнить можно будет остальными тегами.